Source/driver/Castro.cpp

```diff
@@ -1820,9 +1820,9 @@ Castro::computeNewDt (int                   finest_level,
 
 void
 Castro::computeInitialDt (int                   finest_level,
-                          int                   sub_cycle,
+                          int                   /*subcycle*/,
                           Vector<int>&           n_cycle,
-                          const Vector<IntVect>& ref_ratio,
+                          const amrex::Vector<amrex::IntVect>& /*ref_ratio*/,
```

Comment:
The compiler complained about Vector in this line but not in the previous line?

Reply:
Oh, I probably misunderstood the issue the first time around

Reply:
I removed the amrex::